src/ipc-handlers.ts

```diff
@@ -119,11 +121,16 @@ export async function importSite(
 			sendIpcEventToRendererWithWindow( parentWindow, 'on-import', data, id );
 		};
 		const result = await importBackup( backupFile, site.details, onEvent, defaultImporterOptions );
+
+		bumpStat( STATS_GROUP.STUDIO_IMPORT, STATS_METRIC.SUCCESS );
+		bumpStat( STATS_GROUP.STUDIO_IMPORT, result.importerType || 'unknown' );
```

Comment:
Do we intentionally use the same group name for both these calls?

Reply:
I did.
I wasn't sure about the best approach for this, should we have everything regarding import in the same group and track `success`, `failure` and `type`, or have a distinct group for each, like `import_result` and `import_type`. What do you think it's the best?